elab_bridge/Merge.py

```diff
@@ -0,0 +1,178 @@
+        group['id'] = indice
+        matched_indices[old_id] = indice
+        indice += 1
+
+    for field in extrat_fields_sorted.values():
+        if 'group_id' in field and field['group_id'] in matched_indices:
+            field['group_id'] = matched_indices[field['group_id']]
+
+    return extra_fields_groups_sorted, extrat_fields_sorted
+
+
+def merge_jsonfiles(jsonfile_list_sorted: List[str], json_output: str, dry_run: bool = False,
+                    compact: bool = False):
+    """
+    Merge multiple JSON files into one.
+    Args:
+        jsonfile_list_sorted (List[str]): List of JSON files to merge.
+        json_output (str): Output file path.
+        dry_run (bool): If True, preview changes without saving.
+        compact (bool): If True, save output in compact format.
+    """
+    base_data = safe_load_json(jsonfile_list_sorted[0])
+    main_extra_fields_groups = base_data['elabftw']['extra_fields_groups']
+    main_extrat_fields = base_data['extra_fields']
+    indice = get_indices(main_extra_fields_groups)
+
+    for jsonfile in tqdm(jsonfile_list_sorted[1:], desc="Merging JSON files"):
+        extra_fields_groups, extrat_fields = get_a_jsonfile_structure(jsonfile)
+        updated_groups, updated_fields = replace_id(extra_fields_groups, extrat_fields, indice)
+        main_extra_fields_groups.extend(updated_groups)
+        main_extrat_fields.update(updated_fields)
+        indice = max(group['id'] for group in main_extra_fields_groups) + 1
+
+    base_data['elabftw']['extra_fields_groups'] = main_extra_fields_groups
+    base_data['extra_fields'] = main_extrat_fields
+
+    if dry_run:
+        logger.info("Dry run mode enabled. Changes not saved.")
+        logger.info(json.dumps(base_data, indent=4 if not compact else None))
+    else:
+        with open(json_output, 'w') as f:
+            json.dump(base_data, f, indent=None if compact else 4)
+        logger.info(f"Output written to {json_output}")
+
+
+def main():
+    # Argument parser initialization
+    parser = argparse.ArgumentParser(
+        description=(
+            "This script merges multiple JSON files into a single output file. "
+            "Provide a list of JSON files and the name of the output file."
+        ),
+        epilog=(
+            "Examples:\n"
+            "  python Merge.py file1.json file2.json output.json\n"
+            "  python Merge.py file1.json file2.json output.json --dry-run\n"
+            "  python Merge.py file1.json file2.json output.json --overwrite\n\n"
+            "Options:\n"
+            "  --dry-run   Preview the result without saving.\n"
+            "  --compact   Save the output JSON in a compact format (without extra spaces).\n"
+            "  --overwrite Overwrite the output file if it already exists."
+        ),
+        formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.add_argument("--sorted_list_input_files", nargs="+", help="List of JSON files to merge", type=str,
```

Comment:
as I told you, there should be options with one dash and one letter, for example -s

Reply:
this can be done later... please create an issue on the repo to remember that you have to do this later!

Reply:
I'm waiting for you to create this issue before we can merge this PR (so that we don't forget!!!)